server/controllers/authController.js

```diff
@@ -93,15 +93,15 @@ class AuthController {
     try {
       const { rows } = await db.query(queries.selectUser, [req.body.email]);
       if (rows.length === 0) {
-        return res.status(550).send({
+        return res.status(400).send({
           status: res.statusCode,
-          error: 'User with that email is not found',
+          error: 'Invalid email',
```

Comment:
Retain the previous error message as it was more user friendly

Reply:
Ok